doc/api/stream.md

```diff
@@ -90,60 +97,224 @@ server.listen(1337);
 // error: Unexpected token o
 ```
 
-### Class: stream.Duplex
+[Writable][] streams (such as `res` in the example) expose methods such as
+`write()` and `end()` that are used to write data onto the stream.
 
-Duplex streams are streams that implement both the [Readable][] and
-[Writable][] interfaces.
+[Readable][] streams use the [`EventEmitter`] API for notifying application
+code when data is available to be read, or has been read, off the stream.
```

Comment:
I don't think there's an event for "has been read"? (The `"data"` event is kind of an uncomfortable fit there — there's a disconnect between "has been read from underlying resource" and the event; plus listening to the event has the weird property of starting to exhaust the stream.)

Notably, `Writable` streams also use the `EventEmitter` API. Should we mention that here?